src/main/java/com/teamc/fms/controller/FacilityController.java

```diff
@@ -40,4 +42,5 @@ public FacilityDto addFacility(@RequestBody CreateFacilityDto createFacilityDto)
         return facilityMapper.toFacilityDto(facility);
     }
 
+
```

Comment:
I think the changes in this class is useless
two imports not used and a new line
So we can remove them